p2p/host.go

```diff
@@ -88,14 +94,18 @@ func NewHost(self *Peer, key libp2p_crypto.PrivKey) (Host, error) {
 
 	self.PeerID = p2pHost.ID()
 	subLogger := utils.Logger().With().Str("hostID", p2pHost.ID().Pretty()).Logger()
+
+	newMetrics := libp2p_metrics.NewBandwidthCounter()
```

Comment:
make this as an option param to node.sh? Might want to avoid having "all nodes" logging this metric.

Reply:
this is local log, should be fine.

Reply:
ok.